Resources/config/routing.yml

```diff
@@ -91,13 +91,24 @@ canal_tp_mtt_line_timetable_list_defaults:
         seasonId: -?\d+
 
 canal_tp_mtt_line_timetable_list:
-    pattern: /line-timetable/networks/{externalNetworkId}/list/season/{seasonId}/line/{externalLineId}
-    defaults: { _controller: CanalTPMttBundle:LineTimetable:list, externalLineId : null }
+    pattern: /line-timetable/networks/{externalNetworkId}/line/{externalLineId}/list/season/{seasonId}
+    defaults: { _controller: CanalTPMttBundle:LineTimetable:list, seasonId : null }
```

Comment:
I think it lacks externalLineId: null
because here https://github.com/CanalTP/MttBridgeBundle/pull/8/files#diff-f744bebd626ae5268ab11bc80db58d67R138 you don't specify externalRouteId